pkgs/top-level/all-packages.nix

```
@@ -18172,7 +18172,9 @@ with pkgs;
 
   libantlr3c = callPackage ../development/libraries/libantlr3c {};
 
-  libaom = callPackage ../development/libraries/libaom { };
+  libaom = callPackage ../development/libraries/libaom {
+    libjxl = libjxl.override { enableDocs = false; }; # libavif -> libaom -> libjxl -> graphviz -> gd -> libavif
```

```suggestion
    # escape dependency chain with libavif
    libjxl = libjxl.override { enableDocs = false; };
```
Lets not hardcode the path as that might change in the future